CONTRIBUTING.md

```diff
@@ -0,0 +1,58 @@
+# Contributing to Jakarta Concurrency
+
+Thanks for your interest in this project.
+
+## Project description
+
+Jakarta Concurrency provides a specification document, API and TCK for using concurrency from application components without compromising container integrity while still preserving the Jakarta EE platform's fundamental benefits.
+
+ * https://projects.eclipse.org/projects/ee4j.cu
+
+## Developer resources
+
+Information regarding source code management, builds, coding standards, and
+more.
+
+ * https://projects.eclipse.org/projects/ee4j.cu/developer
+
+The project maintains the following source code repositories
+
+ * https://github.com/jakartaee/concurrency
+ * https://github.com/eclipse-ee4j/concurrency-ri
+
+## Eclipse Contributor Agreement
+
+Before your contribution can be accepted by the project team contributors must
+electronically sign the Eclipse Contributor Agreement (ECA).
+
+ * http://www.eclipse.org/legal/ECA.php
+
+Commits that are provided by non-committers must have a Signed-off-by field in
```

Comment:
No.

Reply:
This is the standard process defined by Eclipse Foundatione and adopted by Jakarta EE specifications.

Reply:
It's [no longer needed](https://www.eclipse.org/lists/eclipse.org-committers/msg01291.html) to sign off commits.

Reply:
Yes, you were right. I have removed the sentences. I guess the contribution.md from other Jakarta EE projects need to be updated to reflect the current process.